pkg/cmd/pulumi-language-java/main.go

```diff
@@ -483,3 +485,7 @@ func (host *javaLanguageHost) About(ctx context.Context, req *emptypb.Empty) (*p
 		Metadata:   metadata,
 	}, nil
 }
+
+func (host *javaLanguageHost) RunPlugin(*pulumirpc.RunPluginRequest, pulumirpc.LanguageRuntime_RunPluginServer) error {
```

Comment:
If you embed pulumirpc.UnimplemetnedLanguageServer into javaLanguageHost it'll stop complaining about this and any other new grpc methods we add.